collect_app/src/androidTest/java/org/odk/collect/android/feature/formentry/FormHierarchyTest.java

```diff
@@ -30,6 +30,7 @@
             .around(new CopyFormRule("formHierarchy2.xml", null))
             .around(new CopyFormRule("formHierarchy3.xml", null))
             .around(new CopyFormRule("repeat_group_new.xml", null))
+            .around(new CopyFormRule("Empty First Repeat.xml", null))
```

Comment:
Just a note to myself to rework this test file to remove `CopyFormRule` before merging this. I won't make you slog through that @dimwight.

Reply:
Probably a good thing as my code here is pure copy and paste, I've no idea as to how it works.